src/EFCore.PG/Storage/Internal/Mapping/NpgsqlRegdictionaryTypeMapping.cs

```diff
@@ -0,0 +1,24 @@
+using JetBrains.Annotations;
+using Microsoft.EntityFrameworkCore.Storage;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Utilities;
+using NpgsqlTypes;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.Storage.Internal.Mapping
+{
+    public class NpgsqlRegdictionaryTypeMapping : NpgsqlTypeMapping
+    {
+        public NpgsqlRegdictionaryTypeMapping() : base("regdictionary", typeof(uint), NpgsqlDbType.Oid) { }
```

Comment:
I'm not sure if I should go to the Npgsql source code and add Regdictionary to the NpgsqlDbType enum or just use the Oid.

Reply:
Answered below, thanks!